### ISSUE TITLE
Hyper parameters and Training.

### ISSUE THREAD

**Issue by gevana** [mrharicot/monodepth#142]
HI Clément !

Congrats on the excellent work and thanks for sharing all this code!

I am trying to reproduce your results from scratch , so far I am having problems with "lr_loss_weight" when I assign it with a value (even 0.01) it almost immediately drives all disparity maps (both for left and right) to a constant DC value (which makes lr_loss->0 and disp_gradient_loss->0) .
I see in the both the paper and the code you set it to 1 .

do you have any advice/ insight on this behaviour ? 

Thanks again.
Nadav

**Comment by mrharicot** [comment]
Hi Nadav,

Without seeing your code it is very difficult for me to figure out what is wrong.
Is it working without the left-right consistency?
Are you sure that you are doing the loss between the correct targets?
https://github.com/mrharicot/monodepth/blob/master/monodepth_model.py#L326
It is easy to get confused, even for me after looking at this code so many times.

**Comment by gevana** [comment]
Hi 
Thanks for your help,
Yes, it is working without the left-right consistency although I can't reach the results you reported.
I will continue looking for the bug :) , but can you please confirm the hyper params value you used are: 
alpha_image_loss = 0.85
disp_gradient_loss_weight = 0.1
lr_loss_weight=1.0

Thanks again!

**Comment by mrharicot** [comment]
These numbers seem correct.
You can see them here:
https://github.com/mrharicot/monodepth/blob/master/monodepth_main.py#L40
